pkg/execution/driver/httpdriver/parse.go

```diff
@@ -171,5 +171,5 @@ func parseResponse(byt []byte) any {
 		}
 	}
 
-	return json.RawMessage(byt)
+	return string(byt)
```

Comment:
`json.RawMessage` is just a type alias for `[]byte`, so I think we can just return `byt` as it instead of casting it to a string.